buildroot-external/configs/odroid_m1_defconfig

```diff
@@ -137,7 +137,7 @@ BR2_TARGET_UBOOT_CUSTOM_GIT=y
 BR2_TARGET_UBOOT_CUSTOM_REPO_URL="https://github.com/u-boot/u-boot.git"
 BR2_TARGET_UBOOT_CUSTOM_REPO_VERSION="v2024.01"
 BR2_TARGET_UBOOT_BOARD_DEFCONFIG="odroid-m1-rk3568"
-BR2_TARGET_UBOOT_CONFIG_FRAGMENT_FILES="$(BR2_EXTERNAL_HASSOS_PATH)/bootloader/uboot.config"
+BR2_TARGET_UBOOT_CONFIG_FRAGMENT_FILES="$(BR2_EXTERNAL_HASSOS_PATH)/bootloader/uboot.config $(BR2_EXTERNAL_HASSOS_PATH)/board/hardkernel/odroid-m1/uboot.config"
```

Comment:
Adding it now disables/enables some configs which I am not sure we actually want/need to change. I think rather dropping that config would probably be the better option :thinking: 

FWIW, I just noticed this too: I am seeing Ethernet issues with U-Boot 2024.01 here, and work on a patch which disables Ethernet altogether. For this I need a boards specific config fragment, that is where I've noticed it :sweat_smile:

I suggest we leave this config as is in this PR, and I'll fix it up with the upcoming Ethernet fix.

Reply:
ok I have dropped uboot fragment patch from PR